### PR TITLE
fix: walrus image tag in walrus-images.txt

### PR DESCRIPTION
**Pull request by shanewxy**
**Problem:**

The walrus image tag in walrus-images.txt is still v0.4.0.

**Solution:**

Change to v0.5.0.

**Related Issue:**
#2080

### CODE REVIEW

File: hack/mirror/walrus-images.txt [hunk 1]
```diff
@@ -11,4 +11,4 @@ docker.io/sealio/casdoor:v1.344.0-seal.1
 docker.io/sealio/terraform-deployer:v1.5.7-seal.1
 docker.io/sealio/opentofu-deployer:v1.6.0-seal.1
 docker.io/sealio/hermitcrab:v0.1.3
-docker.io/sealio/walrus:v0.4.0
+docker.io/sealio/walrus:v0.5.0
```
Comment (gitlawr) [inline hack/mirror/walrus-images.txt:14]: When we release v0.5.1, it becomes out of date again. Can we handle it in the CI script?

**Comment by thxCode** [comment]
if so, can we consider with https://github.com/seal-io/walrus/pull/2068#issuecomment-1916625642 to make `PACKAGE_BUILD=false make package` preparing all assets into a specific dir and then upload them?

**Review by thxCode — APPROVED** [review]
LGTM

**Review by aiwantaozi — APPROVED** [review]
LGTM

**Pull request merged**